x-pack/test/apm_api_integration/basic/tests/transactions/transactions_groups_metrics.ts

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import url from 'url';
+import { FtrProviderContext } from '../../../../common/ftr_provider_context';
+import archives from '../../../common/fixtures/es_archiver/archives_metadata';
+
+export default function ApiTest({ getService }: FtrProviderContext) {
+  const supertest = getService('supertest');
+  const esArchiver = getService('esArchiver');
+
+  const archiveName = 'apm_8.0.0';
+  const { start, end } = archives[archiveName];
+
+  const transactionNames = [
+    'DispatcherServlet#doGet',
+    'APIRestController#customers',
+    'APIRestController#order',
+    'APIRestController#stats',
+    'APIRestController#customerWhoBought',
+    'APIRestController#customer',
+    'APIRestController#topProducts',
+    'APIRestController#orders',
+    'APIRestController#product',
+    'ResourceHttpRequestHandler',
+    'APIRestController#products',
+    'DispatcherServlet#doPost',
+  ].join();
+
+  describe('Transactions groups Metrics', () => {
+    describe('when data is not loaded', () => {
+      it('handles the empty state', async () => {
+        const response = await supertest.get(
+          url.format({
+            pathname: `/api/apm/services/opbeans-java/transactions/groups/metrics`,
+            query: {
+              start,
+              end,
+              uiFilters: '{}',
+              numBuckets: 20,
+              latencyAggregationType: 'avg',
+              transactionType: 'request',
+              transactionNames,
+            },
+          })
+        );
+
+        expect(response.status).to.be(200);
+        expect(response.body).to.eql({});
+      });
+    });
+
+    describe('when data is loaded', () => {
```

Comment:
IMO there should be more here than just snapshot tests. Otherwise updating the archive will require more manual inspection.